src/routes/Project/containers/ProjectParamsContainer.js

```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux';
+
+import ProjectParams from '../components/ProjectParams';
+import { updateTempo, updateTimeSig, updateTimeDiv } from '../actions/TableActions';
+
+const mapDispatchToProps = {
+  updateTempo,
+  updateTimeSig,
+  updateTimeDiv
```

Comment:
i feel that, but since this component has to share that state with the table component redux state is the way to go i think. we can always explicitly exclude this from the sync reducer.